symplyphysics/laws/optics/interference_films_from_angle_refraction_refractive_index_wavelength.py

```diff
@@ -0,0 +1,61 @@
+from sympy import (Eq, solve, cos)
+from symplyphysics import (
+    units,
+    Quantity,
+    Symbol,
+    print_expression,
+    validate_input,
+    validate_output,
+    dimensionless,
+    angle_type
+)
+
+# Description
+## Interference in thin films is a phenomenon that occurs as a result of the separation of a light beam when
+## reflected from the upper and lower boundaries of a thin film. As a result, there are two light waves that
+## can interfere.
+## The order of interference indicates the number of whole wavelengths. When the difference in the path of the
+## beam reflected from the upper boundary of the film and the beam reflected from the lower boundary of the
+## film is equal to an integer wave "n", then these two waves enter the observation point with the same phases
+## and demonstrate interference.
```

Comment:
It is wrong. Same phase is not necessary condition to observe interference.

Reply:
> It is wrong. Same phase is not necessary condition to observe interference.

Single-phase and antiphase waves are literally what makes the interference pattern, because single-phase waves add up to each other, forming intensity maxima, and antiphase waves add up to each other, forming minima.

Reply:
Yes, but pattern itself has many phase shifts. When you look at certain point, it appears after some phase shift. It may be minimum, maximum or something in the middle. So your law is about minimum condition that is about certain point of interference pattern.

Reply:
> So your law is about minimum condition that is about certain point of interference pattern.

Yes. And at this point, the waves reflected from the first and second surfaces of the film add up in antiphase, creating an interference minimum.

Reply:
So you law is about interference minimum. Which is not reflected neither in law name nor in the description.